test/test.js

```diff
@@ -0,0 +1,233 @@
+'use strict';
+
+var gulp = require('gulp');
+var path = require('path');
+var should = require('should');
+var assert = require('stream-assert');
+var gutil = require('gulp-util');
+var sourcemaps = require('gulp-sourcemaps');
+var pleeease = require('../index.js');
+
+var fixtures = function (filename) {
+  var join = function (p) {
+    return path.join('./test/fixtures/', p);
+  };
+  if (Array.isArray(filename)) {
+    filename.map(join);
+  }
+  return join(filename);
+};
+
+describe('gulp-pleeease', function () {
+
+  describe('process', function () {
+
+    it('throw error when no files', function (done) {
+      gulp.src('nope')
+        .pipe(pleeease())
+        .on('error', function (err) {
+          err.message.should.eql('Files not found');
+          done();
+        });
+    });
+
+    it('one file without options', function (done) {
+      gulp.src(fixtures('simple.css'))
+        .pipe(pleeease())
+        .pipe(assert.length(1))
+        .pipe(assert.first(function (d) {
+          d.relative.should.eql('app.min.css');
+        }))
+        .pipe(assert.end(done));
+    });
+
+    it('one file with options', function (done) {
+      gulp.src(fixtures('simple.css'))
+        .pipe(pleeease({
+          out: 'out.css'
+        }))
+        .pipe(assert.length(1))
+        .pipe(assert.first(function (d) {
+          d.relative.should.eql('out.css');
+        }))
+        .pipe(assert.end(done));
+    });
+
+    it('multiple file', function (done) {
+      gulp.src(fixtures('*.css'))
+        .pipe(pleeease())
+        .pipe(assert.length(1))
```

Comment:
I don't find any use-cases where someone want to process files separately. This is the main purpose of pleeease, right ? Can you explain ?